godel_plugins/include/godel_plugins/widgets/scan_tool_configuration_window.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SCAN_TOOL_CONFIGURATION_WINDOW_H
+#define SCAN_TOOL_CONFIGURATION_WINDOW_H
+
+#include <godel_plugins/widgets/parameter_window_base.h>
+#include <ui_scan_tool_configuration_window.h>
+#include <godel_msgs/ScanPlanParameters.h>
+
+
+namespace godel_plugins
+{
+
+class ScanPlanConfigWidget : public ParameterWindowBase
+{
+  Q_OBJECT
+public:
+  ScanPlanConfigWidget(godel_msgs::ScanPlanParameters params);
+  godel_msgs::ScanPlanParameters& params() { return params_; }
+  static QStringList quality_metric_list;
```

Comment:
I've never been particularly fond of this code. We need a QStringList to populate a drop-down menu for surface roughness metric. I can hide it in the .cpp code though.